### PR TITLE
[doc] add notes on worker threads

### PR DESCRIPTION
**Pull request by kimula**
### Motivation

The document of Node.js client lacks information on worker threads.

### Modifications

Add notes on worker threads to the document.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

### Documentation

- [x] `doc`

### CODE REVIEW

**Review by nkurihar — APPROVED** [review]

**Review by hrsakai — APPROVED** [review]

File: site2/docs/client-libraries-node.md [hunk 1]
```diff
@@ -291,6 +291,13 @@ const consumer = await client.subscribe({
 });
 ```
 
+> #### Note
+> Pulsar Node.js client uses [AsyncWorker](https://github.com/nodejs/node-addon-api/blob/main/doc/async_worker.md); asynchronous operations such as creating consumers/producers and receiving/sending messages are performed in worker threads.
```
Comment (Anonymitaet) [inline site2/docs/client-libraries-node.md:295]:
```suggestion
> Pulsar Node.js client uses [AsyncWorker](https://github.com/nodejs/node-addon-api/blob/main/doc/async_worker.md). Asynchronous operations such as creating consumers/producers and receiving/sending messages are performed in worker threads.
```

**Review by Anonymitaet — COMMENTED** [review]
@Technoboy- can you review this PR from the tech perspective? Thanks

**Comment by Anonymitaet** [comment]
@kimula 

You delete the backticks (``) so that Bot can not recognize the info and then labels the PR with `doc-info-missing`.
[x] `doc-required`  ✅
[x] doc-required ❌

Instructions of doc labels: https://docs.google.com/document/d/1Qw7LHQdXWBW9t2-r-A7QdFDBwmZh6ytB4guwMoXHqc0/edit#bookmark=id.5d66olk7l4oz

**Review by Technoboy- — APPROVED** [review]

**Comment by Technoboy-** [comment]
> @Technoboy- can you review this PR from the tech perspective? Thanks

**Pull request merged**